servers/physics/body_sw.cpp

```diff
@@ -261,6 +264,10 @@ void BodySW::set_mode(PhysicsServer::BodyMode p_mode) {
 			_inv_mass = mass > 0 ? (1.0 / mass) : 0;
 			_set_static(false);
 		} break;
+		case PhysicsServer::BODY_MODE_SOFT: {
+			// Only implemented in Bullet physics backend
+			ERR_PRINT("BODY_MODE_SOFT is only implemented in the Bullet physics backend.");
+		} break;
```

Comment:
```
servers/physics/body_sw.cpp:236:9: warning: enumeration value 'BODY_MODE_SOFT' not handled in switch [-Wswitch]
servers/physics/body_sw.cpp:58:9: warning: enumeration value 'BODY_MODE_SOFT' not handled in switch [-Wswitch]
```
@AndreaCatania Am I correct in my understanding that `body_sw.cpp` is the GodotPhysics backend and that the Soft mode doesn't make sense there?

Reply:
Resolving, further discussion in https://github.com/godotengine/godot/pull/22447/files#r220975694.